### PR TITLE
Update docker-compose installation instructions on Arch Linux

### PR DESCRIPTION
**Pull request by inpv**
### Motivation and context

Describes fixes for `docker-compose` installation (and container creation) on Arch Linux, see this [issue](https://github.com/opencv/cvat/issues/5214).

### How has this been tested?

Tested on a (custom) Arch Linux dev env, using docker 20.10.21 and docker-compose 1.29.2.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/contributing/development-environment.md [hunk 1]
```diff
@@ -167,6 +167,25 @@ description: 'Installing a development environment for different operating syste
...
+  > ```bash
+  > pikaur -S docker-compose-v1-bin
+  > ```
+  > You will also need to either additionally install `libxcrypt-compat`
+  > ```bash
+  > sudo pacman -S libxcrypt-compat
+  > ```
+  > or create a symlink between the newer and older `libcrypt.so` libraries
+  > ```bash
+  > sudo ln -s /usr/lib/libcrypt.so /usr/lib/libcrypt.so.1
```
[the remark this answers is not in the document]
Comment (inpv) [reply site/content/en/docs/contributing/development-environment.md:183]: Well, I got the info about version difference from this [question](https://stackoverflow.com/questions/69464001/docker-compose-container-name-use-dash-instead-of-underscore) as well as the official docker [changelog](https://docs.docker.com/compose/release-notes/#280).

However, I did test version 2 with the lib fixes and the containers spawned just fine, so I'll correct the docs about the version part.

**Comment by inpv** [comment]
@zhiltsov-max Removed the version part from the docs.

**Comment by zhiltsov-max** [comment]
I still can see these changes speaking about docker. The problem is not related to docker at all. As you can see in [the error message](https://github.com/opencv/cvat/issues/5214#issuecomment-1299921019), the error appears in one of the rest api tests. I suggest to add these changes to the installation section [here](https://github.com/opencv/cvat/blob/develop/site/content/en/docs/contributing/development-environment.md?plain=1#L132-L146).

**Comment by inpv** [comment]
True, my mistake. I guess this PR should be closed without merging because it was only my local issue when executing `test_load_register.py`, which comes from another branch altogether. I launched the tests without these lib fixes and the only errors I got was that the containers are already running:
```
(.env) [%USER%@%HOST% cvat]$ python3.9 -m pytest tests/python
===================================== test session starts ======================================
platform linux -- Python 3.9.15, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- ~/Development/cvat/.env/bin/python3.9
cachedir: .pytest_cache
rootdir: ~/Development/cvat/tests/python, configfile: pytest.ini
plugins: Faker-15.1.3
collected 559 items                                                                            

tests/python/cli/test_cli.py::TestCLI::test_can_create_task_from_local_images 

! _pytest.outcomes.Exit: It's looks like you already have running cvat containers. Stop them and try again. List of running containers: nuclio-local-storage-reader, test_minio_1, cvat_ui, cvat_worker_default, cvat_server, cvat_worker_low, cvat_opa, cvat_worker_webhooks, cvat_utils, cvat_redis, nuclio, cvat_db, traefik, test_logstash_1, test_cvat_utils_1, test_cvat_ui_1, test_webhook_receiver_1, test_cvat_worker_default_1, test_cvat_worker_low_1, test_cvat_server_1, test_cvat_worker_webhooks_1, test_kibana_1, test_cvat_db_1, test_cvat_redis_1, test_elasticsearch_1 !
==================================== no tests ran in 0.99s =====================================
```
After running command `docker kill $(docker ps -q)` the tests launch fine.

I will also update this [comment](https://github.com/opencv/cvat/issues/5214#issuecomment-1299921019) with the false alarm.

**Comment by inpv** [comment]
Closing the PR for now - doesn't have to do with `docker-compose` versions on Arch.

**Pull request closed without merging**